src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -12,8 +12,8 @@
     <SharedFrameworkHostFileNameOverride>crossgen2</SharedFrameworkHostFileNameOverride>
     <!-- Build this pack for any RID if building from source. Otherwise, only build select RIDs. -->
     <RuntimeIdentifiers Condition="'$(DotNetBuildSourceOnly)' != 'true'">linux-x64;linux-musl-x64;linux-arm;linux-musl-arm;linux-arm64;linux-musl-arm64;osx-x64;osx-arm64;win-x64;win-x86;win-arm64</RuntimeIdentifiers>
-    <!-- FreeBSD runtime/apphost packs aren't built in the official build so only reference the RIDs when targetting FreeBSD -->
-    <RuntimeIdentifiers Condition="'$(DotNetBuildSourceOnly)' != 'true' and '$(TargetOS)' == 'freebsd'">$(RuntimeIdentifiers);freebsd-x64;freebsd-arm64</RuntimeIdentifiers>
+    <!-- runtime/apphost packs of these platforms aren't built in the official build, so only reference the RIDs when we are targetting the community-supported platforms. -->
+    <RuntimeIdentifiers Condition="'$(DotNetBuildSourceOnly)' != 'true' and ('$(TargetOS)' == 'freebsd' or '$(TargetArchitecture)' == 'riscv64')">$(RuntimeIdentifiers);freebsd-x64;freebsd-arm64;linux-riscv64;linux-musl-riscv64</RuntimeIdentifiers>
```

Comment:
please keep those two cases split, otherwise a standalone FreeBSD build or a standalone riscv64 build will run into the same issue.

Reply:
The original issue for which you split them last week was about vertical builds which only happens in official builds. It is not something which can happen in standalone mode.

Reply:
I agree with Alexander, these should be split. We don't want to build the freebsd RID specific packages when targeting riscv64 and vice versa.